thirdparty/deto_async/async/changednotify.h

```diff
@@ -163,20 +163,20 @@ class ChangedNotify
         {
             CallType type = static_cast<CallType>(_type);
             switch (type) {
-            case Undefined: {} break;
-            case Changed:     {
+            case CallType::Undefined: {} break;
```

Comment:
since when do we put braces in `case` statements?

Reply:
yes, sorry the question wasn't to you in particular. Anyway, it does no harm.

Reply:
Just noticed: this is third party code!